src/js/common/schemaform/save-in-progress/RoutedSavableApp.jsx

```diff
@@ -102,7 +102,7 @@ class RoutedSavableApp extends React.Component {
       newProps.setFetchFormStatus(LOAD_STATUSES.notAttempted);
     } else if (newProps.prefillStatus !== this.props.prefillStatus
       && newProps.prefillStatus === PREFILL_STATUSES.unfilled) {
-      newProps.router.push(newProps.routes[this.props.routes.length - 1].pageList[0].path);
+      newProps.router.push(newProps.routes[this.props.routes.length - 1].pageList[1].path);
```

Comment:
This is a bug I noticed when there's an error prefilling. We actually want to start the form, which is the second page (intro page is first).